### ISSUE TITLE
Deconvolution accuracy

### ISSUE THREAD

**Issue by lventurasp** [li-lab-mcgill/gtm-decon#5]
Hi!! I am trying to evaluate the deconvolution accuracy with simulated data. I made a code on my own to sum up all the gene counts in the test set of the reference single-cell dataset and then perform the deconvolution of it. However, I am not sure if I am proceeding properly. Could you share the simulate_bulkRS.py code, please? 
Another thing, if we are using the training set for training the model and the test to test the accuracy, why are we also separating a validation set? Thanks a lot!
Best regards,
Laura.

**Comment by aosakwe** [comment]
Hi @lventurasp ! Thank you for your interest in using gtm-decon. Based on your description, the simulation process you are using is correct. The simulation step randomly samples a set of single cell expression profiles and combines them into a pseudo-bulked sample. So long as the sampled single cells used were labelled you can then create a ground truth cell type proportion estimate for the simulated bulk data.

We also create a validation set to be able to finetune the hyper-parameters of the gtm-decon model (# of topics per cell type). Testing different instances of the model on the validation set will allow the user to select the instance with the best parameterization prior to applying it to the test set. If you are not finetuning the hyper-parameters you can also comment out the validation set code in train_test_validation_split.py and change line 83 to be ```num_training = int(len(X) * 0.8)```
